loudness/patterns/front-page.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Title: Front Page
+ * Slug: loudness/front-page
+ */
+?>
+
+<?php
+if ( 'posts' === get_option( 'show_on_front' ) ) {
```

Comment:
I think this is quite clever, and it works. But is it ultimately a workaround that should be addressed in core? cc @scruffian @MaggieCabrera for their thoughts.

The problem we are trying to solve is `home.html` isn't working well for home page setup, especially on dotcom. Lots of context here https://dotcom.wordpress.com/2022/09/01/template-vs-post-page-editing-the-confusion and https://themedevp2.wordpress.com/2022/09/20/unified-approach-to-home-page-setup-across-themes/

The idea with this PR is that we supply a front-page instead of home, so there is one template used for the home page. Depending on the site's reading settings, we would like it to render different blocks, by default.